include/SANDTrackerPlane.h

```diff
@@ -0,0 +1,105 @@
+#pragma once
+#include "SANDTrackerCell.h"
+#include <vector>
+#include <map>
+class SANDTrackerModule;
```

Comment:
Why not defining some namespaces to keep the Class name shorter, as for instance:

`sand::tracker::Module`

namespaces have pros and cons but brings benefit both to readibility and scoping, as suggested here: https://google.github.io/styleguide/cppguide.html#Namespaces